docs/pages/modules/config-plugin-and-native-module-tutorial.mdx

```diff
@@ -181,13 +220,13 @@ At the root of your project, run `npm run build plugin` to start the TypeScript
 }
 ```
 
-Now when running `npx expo prebuild` inside our **example** folder we should see our 'my custom plugin' console.log statement in the terminal.
+When you run `npx expo prebuild` inside your **example** folder, you see your 'my custom plugin' console.log statement in the terminal.
 
 <Terminal cmd={['$ cd example', '$ npx expo prebuild --clean']} />
 
-To inject our custom API keys into **AndroidManifest.xml** and **Info.plist** we can use a few helper [`mods` provided by `expo/config-plugins`](/config-plugins/plugins-and-mods/#what-are-mods), which makes it easy to modify native files. In our example, we will use two of them, `withAndroidManifest` and `withInfoPlist`.
+To inject your custom API keys into **AndroidManifest.xml** and **Info.plist**, use helper [`mods` provided by `expo/config-plugins`](/config-plugins/plugins-and-mods/#what-are-mods). These make it easy to modify native files. For this example, use `withAndroidManifest` and `withInfoPlist`.
 
-As the name suggests, `withInfoPlist` allows us to read and modify **Info.plist** values. Using the `modResults` property, we can add custom values as demonstrated in the code snippet below:
+As the name suggests, `withInfoPlist` allows you to read and modify **Info.plist** values. Using the `modResults` property, you can add custom values as shown in the code snippet below:
```

Comment:
Can we move this iOS section after the Android one for platform order consistency as per the writing style guide?